scanner/datastore/postgres/vuln_update.go

```diff
@@ -3,48 +3,81 @@ package postgres
 import (
 	"context"
 	"errors"
-	"fmt"
-	"net/http"
-	"strings"
 	"time"
 
 	"github.com/jackc/pgx/v4"
 )
 
-const lastVulnUpdateKey = `last-vuln-update`
-
-// GetLastVulnerabilityUpdate retrieves the last vulnerability update from the database.
+// GetLastVulnerabilityUpdate implements MatcherMetadataStore.GetLastVulnerabilityUpdate.
 //
-// The returned time will be in the form of http.TimeFormat.
+// Assumes the last update is the oldest update timestamp in the table.
 func (m *matcherMetadataStore) GetLastVulnerabilityUpdate(ctx context.Context) (time.Time, error) {
-	const selectTimestamp = `SELECT timestamp FROM last_vuln_update WHERE key = $1`
-
-	var t string
-	row := m.pool.QueryRow(ctx, selectTimestamp, lastVulnUpdateKey)
+	const selectTimestamp = `
+		SELECT update_timestamp
+		FROM last_vuln_update
+		WHERE update_timestamp IS NOT NULL ORDER BY update_timestamp LIMIT 1;`
+	row := m.pool.QueryRow(ctx, selectTimestamp)
+	var t time.Time
 	err := row.Scan(&t)
 	if errors.Is(err, pgx.ErrNoRows) {
 		return time.Time{}, nil
 	}
 	if err != nil {
 		return time.Time{}, err
 	}
+	return t.UTC(), nil
```

Comment:
Should the caller handle the timezone?

Reply:
This is to ensure the timestamp stored is the same regardless of the the TZ Scanner is running on.

Reply:
I don't quite understand. I thought this was the retrieval of the data. Looks like we're doing the UTC conversion during the insert

Honestly, this feels like a nitpick on my part, so I'll mark it as resolved, but I'd still like to understand if you have time to explain